app/uk/gov/hmrc/gatekeeper/models/User.scala

```diff
@@ -145,3 +145,10 @@ case class Developer(user: User, applications: List[Application], xmlServiceName
 
   lazy val id: String = user.userId.value.toString
 }
+
+case class UserPaginatedResponse(totalCount: Int, users: List[RegisteredUser])
+
+object UserPaginatedResponse {
+  implicit val userFormat: Format[RegisteredUser]    = RegisteredUser.registeredUserFormat
```

Comment:
I think this is not needed. All tests still pass, and I haven't spotted a problem with local testing (yet).

Reply:
Addressed